src/components/Header/AddMovieButton/AddMovieButton.tsx

```diff
@@ -1,7 +1,22 @@
 import styles from "./addMovieButton.module.css";
+import AddModal from "../../Modal/AddModal";
+import { useState } from "react";
```

Comment:
specific imports like `react`, `react-router` must come first 

if interested, there is an eslint plugin for import order